File: python/7 kyu/266.py
# https://www.codewars.com/kata/57eaec5608fed543d6000021/train/python

# Passed

def div_con(x):
    int_total, string_total = 0, 0
    for elem in x:
        if type(elem) == str:
            string_total += int(elem)
        else:
            int_total += elem
            
    result = int_total - string_total
    return result

output = div_con(['5', '0', 9, 3, 2, 1, '9', 6, 7])
print(output)

File: nodejs/8 kyu/266.js
// https://www.codewars.com/kata/57613fb1033d766171000d60/train/javascript

// Passed

function uefaEuro2016(teams, scores) {
  let [team1, team2] = teams;
  let [team1Score, team2Score] = scores;
  let result = `At match ${team1} - ${team2}, `;

  if (team1Score == team2Score) {
    result += "teams played draw.";
  } else {
    if (team1Score > team2Score) {
      result += `${team1} won!`;
    } else {
      result += `${team2} won!`;
    }
  }

  return result;
}

const output = uefaEuro2016(["Belgium", "Italy"], [0, 2]);
console.log(output);